crates/client/src/serialize/txt/rdata_parsers/sshfp.rs

```diff
@@ -0,0 +1,55 @@
+// TODO license
+
+//! SSHFP records for SSH public key fingerprints
+
+use error::*;
+use rr::rdata::SSHFP;
+
+const HEX: ::data_encoding::Encoding = new_encoding! {
```

Comment:
I'm assuming this means that this format isn't currently supported by the data_encoding library?

Reply:
Yes, it is not in data_encoding. It is copy-pasted from [here](https://github.com/bluejekyll/trust-dns/blob/7c8a0739dad495bf5a4fddfe86b8bbe2aa52d060/crates/client/src/serialize/txt/rdata_parsers/tlsa.rs#L16). It could be shared, is there a canonical module in trust_dns for "utility things"?

Reply:
Not as such. I’m guessing there might be enough things around data encoding/base64 where this might be desirable.

Don’t worry about that in this pr though, that seems like something we could clean up later.